extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -37,7 +39,8 @@ export function registerTableDesignerCommands(appContext: AppContext) {
 				id: generateUuid(),
 				connectionString: connectionString,
 				accessToken: context.connectionProfile!.options.azureAccountToken as string,
-				tableIcon: tableIcon
+				tableIcon: tableIcon,
+				additionalInfo: `${context.connectionProfile.connectionName ? '(' + context.connectionProfile.connectionName + ')' : ''}${nonDefaultOptions}`
```

Comment:
You don't need to include Profile name in brackets in additional info, since the tab title already displays that. So just keep it plain and simple.

Reply:
It currently does not in the main build so far, but I added braces to help make it more visible.

Reply:
I decided to use a hyphen after the short title/name. I can also add the connection name to the tab description as done earlier

Reply:
I meant to say you can let connection name be only in the main title, and show connection options with advanced options on hover.